repository/model/template.go

```diff
@@ -0,0 +1,52 @@
+package model
+
+import (
+	svcmodel "release-manager/service/model"
+
+	"github.com/google/uuid"
+)
+
+type Template struct {
+	ID         uuid.UUID      `json:"id"`
+	Type       string         `json:"type"`
+	ReleaseMsg ReleaseMessage `json:"template_data"`
```

Comment:
Calling it `template_data` in the db may not be the best wording, it's quite confusing.